lib/Elevate/Leapp.pm

```diff
@@ -87,6 +96,64 @@ sub upgrade ($self) {
     return;
 }
 
+sub search_report_file_for_blockers ($self) {
+
+    my @blockers;
+    my @ignored_blockers = qw(
```

Comment:
constant at top of file?

Reply:
Since it is only used in this function, I didn't see the need to make it available to a wider scope.  Though I was thinking of refactoring that so the ignored blocker list is passed in.  If we did that, we could refactor _report_leapp_failure_and_die() to use search_report_file_for_blockers() find all the blockers reported as a result of failure.  As it stands it only prints out information on the blocker to do with a problem with the answer file (which we preemptively fix).  It doesn't filter based in the "inhibitor" flag.